web/src/beta/components/Modal/index.tsx

```diff
@@ -49,12 +49,16 @@ const Modal: React.FC<Props> = ({
       }));
       setTabsFields(convertedFields);
     }
-  }, [sidebarTabs]);
+
+    // eslint-disable-next-line react-hooks/exhaustive-deps
```

Comment:
This should only be used if absolutely necessary. Can you look at how you might refactor this component to avoid this issue instead? 
For example, I am not sure setting sidebarTabs to TabsFields is necessary. Instead of local state for tabsfields, which should be static, having local state for what tab is currently selected makes more sense. Or if you can see a better implementation 🙏🏻

Reply:
@kyle sure let me check how to re-implement this